hack_loki.md

```diff
@@ -0,0 +1,94 @@
+# Hacks on Loki & Grafana
+
+## Loki Operator
+
+### From Sources
+
+Clone loki-operator repository and open folder
+```bash
+git clone https://github.com/ViaQ/loki-operator.git
+cd loki-operator
+```
+
+Disable loki auth for development in [loki-config.yaml](https://github.com/ViaQ/loki-operator/blob/master/internal/manifests/internal/config/loki-config.yaml#L2)
+`sed -i 's/auth_enabled: true/auth_enabled: false/g' /internal/manifests/internal/config/loki-config.yaml`
+
+Build and push the container image and then deploy the operator
+`make oci-build oci-push deploy REGISTRY_ORG=netobserv VERSION=latest`
+
+Create a LokiStack instance to get the various components of Loki up and running
+`kubectl apply -f hack/lokistack_dev.yaml`
+
+### From image (WARNING WIP, need kubectl [1.23.0-alpha.4](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md#v1230-alpha4) or more. Issues on /host access in debug ephemeral container)
+
+Get operator from official image and create instance
+```bash
+kubectl apply -f examples/lokioperator.yaml
+kubectl apply -f examples/lokistack_dev.yaml
+```
+
+Get controller manager name
+`kubectl get pods -l name=loki-operator-controller-manager`
+
+Create ephemeral container using controller manager name like :
+`kubectl debug controller-manager-9cb578d85-5dcxg -it --target=manager --image=busybox`
+
+Disable auth in loki config 
+`sed -i 's/auth_enabled: true/auth_enabled: false/g' /host/workspace/internal/manifests/internal/config/loki-config.yaml`
+
+Then restart all loki pods to update configmap
+`kubectl get pods -n default --no-headers=true | awk '/loki-/{print $1}'| xargs  kubectl delete -n default pod`
+
+## Loki & Grafana stack with Helm
+
+Deploy [loki and grafana](https://grafana.com/docs/loki/latest/installation/helm/#deploy-grafana-to-your-cluster) to your cluster from official images using helm
+```bash
+helm upgrade --install loki grafana/loki-stack --set promtail.enabled=false
+helm install loki-grafana grafana/grafana
+```
+
+Wait for the service running then forward port
+`kubectl port-forward service/loki-grafana 3000:3000`
```

Comment:
it seems to be `3000:80`, not `3000:3000`

Reply:
it depends on what you [open next](https://github.com/netobserv/documents/pull/1/files/97ea0041d55a5a4497f6a614a9845d6ed2563655#diff-7f51acaa5430f57ab824677d12e76ad94983d7dd45c949b7cb1335991a019996R78) ?

Reply:
no, if I'm correct it's the opposite, `3000:80` means the service exposes 80 and you expose 3000 on your machine.
I actually followed your step and had an error about port 3000 not being exposed by the service

Reply:
Ok I'll double check that :+1: 
Thanks for the input